lib/internal/Magento/Framework/Option/ArrayInterface.php

```diff
@@ -7,6 +7,7 @@
 
 /**
  * @todo Remove in favor of the ancestor interface
+ * @deprecated
```

Comment:
Would you be able to add a note for what is the preferred option now instead of this interface? This would help users out when developing to see yes this is deprecated and exactly what interface to use instead of this one.